fluss-flink/fluss-flink-2.1/src/test/java/org/apache/fluss/flink/catalog/Flink21CatalogITCase.java

```diff
@@ -17,5 +17,267 @@
 
 package org.apache.fluss.flink.catalog;
 
+import org.apache.flink.table.api.DataTypes;
+import org.apache.flink.table.api.Schema;
+import org.apache.flink.table.catalog.CatalogTable;
+import org.apache.flink.table.catalog.ObjectPath;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.apache.fluss.flink.FlinkConnectorOptions.BUCKET_KEY;
+import static org.apache.fluss.flink.FlinkConnectorOptions.BUCKET_NUMBER;
+import static org.assertj.core.api.Assertions.assertThat;
+
 /** IT case for catalog in Flink 2.1. */
-public class Flink21CatalogITCase extends FlinkCatalogITCase {}
+public class Flink21CatalogITCase extends FlinkCatalogITCase {
+
+    @BeforeAll
+    static void beforeAll() {
+        FlinkCatalogITCase.beforeAll();
+
+        // close the old one and open a new one later
+        catalog.close();
+
+        catalog =
+                new Flink21Catalog(
+                        catalog.catalogName,
+                        catalog.defaultDatabase,
+                        catalog.bootstrapServers,
+                        catalog.classLoader,
+                        catalog.securityConfigs);
+        catalog.open();
+    }
+
+    @Test
+    @Override
+    void testCreateTable() throws Exception {
+        // create a table will all supported data types
+        tEnv.executeSql(
```

Comment:
Why need this test? It feels like this test is not really related to the content of the PR.